src/WorkerChannel.ts

```diff
@@ -2,22 +2,32 @@
 // Licensed under the MIT License.
 
 import { Context } from '@azure/functions';
+import { readJson } from 'fs-extra';
 import { AzureFunctionsRpcMessages as rpc } from '../azure-functions-language-worker-protobuf/src/rpc';
 import { IFunctionLoader } from './FunctionLoader';
 import { IEventStream } from './GrpcClient';
+import path = require('path');
 
 type InvocationRequestBefore = (context: Context, userFn: Function) => Function;
 type InvocationRequestAfter = (context: Context) => void;
 
+export interface PackageJson {
+    type?: 'commonjs' | 'module';
+}
+
 export class WorkerChannel {
     public eventStream: IEventStream;
     public functionLoader: IFunctionLoader;
+    public functionAppDir: string;

```

Comment:
`functionAppDir` is never used outside `initAppDir`. Please remove it as a property

Reply:
I figured it might become useful in the future, but I guess we could always add it if we need it. Removed it for now!